stripper/ze_omochix_v3_2fix.cfg

```diff
@@ -39,3 +39,22 @@ modify:
 		"spawnflags" "1"
 	}
 }
+
+;Nerf boss hp since GFL never gets past it and is probably a bit tight
```

Comment:
We've only played the map 2 times since testing it according to map history, quite a small sample size for "GFL never gets past it".

I'd like to note as well, as just being a config contributor it doesn't give you the power to decide on nerfs like this. This is the kind of thing that stays up to us (or the mappers). If I had expressed wanting to do it or told you it was okay beforehand, then it'd be different.